plugins/inputs/raindrops/raindrops.go

```diff
@@ -16,8 +16,8 @@ import (
 )
 
 type Raindrops struct {
-	Urls        []string
-	http_client *http.Client
+	Urls       []string
```

Comment:
Hmmm `Url` is changed to `URL` but `Urls` is not changed to `URLs`?

Reply:
Revive didn't find plurals. This rule can be configured to find all `Urls` but it will suggest to rename them to `URLS`...
I don't know if we want to follow that path...

Reply:
Not sure but it caught my attention. :-)